components/layouts/mainLayout.tsx

```diff
@@ -30,6 +32,21 @@ export const MainLayout: FC<PropsWithChildren<MainLayoutProps>> = ({
     userType === 'DOCTOR' ? doctorNavigation : patientNavigation
 
   const userName = 'Tom Cook'
+
+  const { mutate } = useMutation({
+    mutationFn: logoutUser,
+    onSuccess: () => {
+      router.push('/login')
+    },
+    onError: (error) => {
+      console.log('error' ,error)
+    }
+  })
+
+  const onLogoutClick = (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
```

Comment:
why the preventDefault?

Reply:
because the sign out button is inside an <a></a> tag that will eventually redirect to the patient information page